frontend/src/views/web/challenge-list.html

```diff
@@ -1,6 +1,30 @@
 <section class="ev-sm-container ev-view challenge-container">
+    <p class="text-med-black fs-30 no-margin">All Challenges</p>
+    <!-- tabs for challenges -->
```

Comment:
When you are on the ongoing challenge tab, the api for all the challenges will be called. Modify it, so that the challenges relevant to the tab are shown